### PR TITLE
[test-env] wrapper

### PR DESCRIPTION
**Pull request by Barackos**

### CODE REVIEW

**Comment by oded-soffrin** [comment]
@lxgreen @shaulgo this is the wrapper test if you want to discuss.
I agree duplicate tests are a pain, but we should at least check themeing for each plugin
do you think anything here is redundant ?

**Comment by shaulgo** [comment]
> @lxgreen @shaulgo this is the wrapper test if you want to discuss.
> I agree duplicate tests are a pain, but we should at least check themeing for each plugin
> do you think anything here is redundant ?

I'm talking with Barak about this. I think we should move most of the tests to wrappers. That way we test both wrapper and rce. Only tests that have a config that isn't supported in wrappers should be left without it.

File: packages/wrapper/web/src/pluginsStrategy/pluginsStrategy.ts [hunk 1]
```diff
@@ -39,15 +39,16 @@ function viewerStrategy(
...
     typeMappers: (typeMapper && prev.typeMappers.concat([typeMapper])) || prev.typeMappers,
     decorators:
       (decorator && prev.decorators.concat([decorator(theme, config)])) || prev.decorators,
     inlineStyleMappers:
       (inlineStyleMapper &&
         contentState &&
-        prev.inlineStyleMappers.concat([inlineStyleMapper?.(config, contentState)])) ||
+        prev.inlineStyleMappers.concat([inlineStyleMapper?.(finalConfig, contentState)])) ||
       prev.inlineStyleMappers,
   };
```
Comment (Barackos) [inline packages/wrapper/web/src/pluginsStrategy/pluginsStrategy.ts:53]: This change was needed to solve custom inlineStyleMappers for the viewer.
Identically solved in pr #983.

**Comment by Barackos** [comment]
This branch has unexpected behaviour in Applitools, tests stopped passing.
Squashed to (#1045)

**Pull request closed without merging**